src/qonlinetranslator.h

```diff
@@ -188,13 +188,15 @@ class QOnlineTranslator : public QObject
     {
         QString example;
         QString description;
+        QJsonObject toJson() const;
```

Comment:
Could you move these classes to separate files? Since they have a function, then I would not mix its code with parsing requests from translation engines (`qonlinetranslator.cpp` is already big enough).

Reply:
Should I change the `struct` keyword to `class` when I do that or leave it be?

Reply:
Let's leave it as `struct` :)